### PR TITLE
Removed active AI focus dithering

### PR DESCRIPTION
**Pull request by Grummel7**
See https://www.freeorion.org/forum/viewtopic.php?p=109993&sid=403d00698660ab7fdeeceb7b52b204a4#p109993

### CODE REVIEW

**Comment by geoffthemedio** [comment]
Provide some rationale for this?

@Cjkjvfnby @Morlic-fo any comment?

File: default/python/AI/ResourcesAI.py [hunk 1]
```diff
@@ -722,59 +721,7 @@ def set_planet_industry_and_research_foci(focus_manager, priority_ratio):
...
-                # calculation prevents the indicator from stopping recovery mode one turn too early.
-                research_capped = (rr - cr) <= 0.5
-                industry_recovery_phase = (ii - ci) - (cr - tr) > AIDependencies.FOCUS_CHANGE_PENALTY - research_penalty
-
-                if not (research_capped or industry_recovery_phase):
-                    target_pp += ci - 1 - research_penalty
-                    target_rp += cr + 1
-                    focus_manager.bake_future_focus(pid, RESEARCH, False)
-                continue
+            # removed dithering, meters now fall too quickly for it to ever be worth it
```
Comment (geoffthemedio) [inline default/python/AI/ResourcesAI.py:724]: a comment about code that is no longer present isn't very useful, I think...

Comment (Grummel7) [reply default/python/AI/ResourcesAI.py:724]: Well, somebody may wonder why there is no step 3.
I guess I will do more work on this code anyway. When using focuses, the AI should also use influence focus.

**Comment by geoffthemedio** [comment]
```
Run flake8 default/python --config=default/python/tox.ini --format="::error file=%(path)s,line=%(row)d,col=%(col)d::%(path)s:%(row)d:%(col)d: %(code)s %(text)s"
Error: default/python/AI/ResourcesAI.py:678:5: F841 local variable 'has_force' is assigned to but never used
Error: default/python/AI/ResourcesAI.py:711:13: F841 local variable 'planet' is assigned to but never used
Error: Process completed with exit code 1.
```

File: default/python/AI/ResourcesAI.py [hunk 2]
```diff
@@ -675,7 +675,8 @@ def set_planet_industry_and_research_foci(focus_manager, priority_ratio):
     target_pp = 0.001
     target_rp = 0.001
     resource_timer.start("Loop")  # loop
-    has_force = tech_is_complete("CON_FRC_ENRG_STRC")
+    # currently unused, but may be needed it again
+    # has_force = tech_is_complete("CON_FRC_ENRG_STRC")
```
Comment (Cjkjvfnby) [inline default/python/AI/ResourcesAI.py:679]: I don't like to have commented code. It looks very abandoned. 

We could just add docstring and something like that:
```
We could consider using CON_FRC_ENRG_STRC for more precise calculations.
```

File: default/python/AI/ResourcesAI.py [hunk 3]
```diff
@@ -701,15 +702,13 @@ def set_planet_industry_and_research_foci(focus_manager, priority_ratio):
     maxi_ratio = cumulative_rp / max(0.01, cumulative_pp)
 
     aistate = get_aistate()
-    for adj_round in [1, 2, 3, 4]:
+    for adj_round in [1, 2, 4]:
```
Comment (Cjkjvfnby) [inline default/python/AI/ResourcesAI.py:705]: You could rename `4 to 3`

**Comment by geoffthemedio** [comment]
Superceded as noted in #3729 description

**Pull request closed without merging**